boggle.js

```diff
@@ -33,12 +33,13 @@ function randomInteger(min, max) {
 function shake(boggleBoard) {
   for (let row of boggleBoard)
   {
-    for (i=0;i<=4;i++)
+    for (i=0;i<4;i++)
     {
       let index = dice[(randomInteger(0, dice.length - 1))];
-      row[i] = index[Math.floor(Math.random() * (7))];
+      row[i] = index[Math.floor(Math.random() * (6))];
```

Comment:
1. No need for parens around 6.
2. Why hard-code `6` here but use `dice.length` in the line above?